DQM/Integration/python/clients/mutracking_dqm_sourceclient-live_cfg.py

```diff
@@ -96,16 +96,35 @@
                                    process.muonCSCDigis +
                                    process.muonDTDigis +
                                    process.muonRPCDigis +
-                                   process.muonRPCNewDigis +
```

Comment:
Comparing this backport with the original PR #28488, I see you are removing muonRPCNewDigis from this sequence. It was already removed in 11_1_X. 
See: https://github.com/cms-sw/cmssw/pull/28488/files#diff-b25c3200c94b7e485b033fb90ab6e51fL99 .
What are the effects of this change?